api/tests/functional/test_api.py

```diff
@@ -0,0 +1,27 @@
+def test_locations_list(test_client):
```

Comment:
These tests depend on the current state of the dev database, which could change over time or from person to person. It would be better to implement a fixture that populates the database with test data and removes all data after each run. This will prevent changes in data from breaking our tests.